### PR TITLE
MTDSA-3548 - template microservice API

### PR DESCRIPTION
**Pull request by jeremystone**
SampleController / Orchestrator / Service with generic Connector.

### CODE REVIEW

File: app/definition/ApiDefinitionFactory.scala [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        )
+      ),
+      api = APIDefinition(
+        name = "Individual Losses (MTD)",
+        description = "An API for providing individual losses data",
+        context = appConfig.apiGatewayContext,
+        versions = Seq(
+          APIVersion(version = VERSION_1, access = buildWhiteListingAccess(), status = buildAPIStatus(VERSION_1), endpointsEnabled = true)
+//          ,
+//          APIVersion(version = VERSION_2, access = buildWhiteListingAccess(), status = buildAPIStatus(VERSION_2), endpointsEnabled = true)
```
Comment (james-work-account) [inline app/definition/ApiDefinitionFactory.scala]: Why is this here? Is it meant to be commented out?

File: conf/application.conf [hunk 2]
```diff
@@ -75,45 +75,47 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.MicroserviceModule"
...
-  }
-
-  uk.gov.hmrc.hello.controllers.LiveController = {
-    needsAuth = false
-    needsLogging = true
-    needsAuditing = false
-  }
-
-}
+//controllers {
```
Comment (james-work-account) [inline conf/application.conf]: Why is this all commented out? Is it not needed?

File: project/CodeCoverageSettings.scala [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    "prod.*",
+    ".*Routes.*",
+    "config.*",
+    "testOnly.*",
+    "testOnlyDoNotUseInAppConf.*"
+  )
+
+  val settings: Seq[Setting[_]] = Seq(
+    ScoverageKeys.coverageExcludedPackages := excludedPackages.mkString(";"),
+    ScoverageKeys.coverageMinimum := 95,
```
Comment (james-work-account) [inline project/CodeCoverageSettings.scala]: `sbt clean coverage test it:test coverageReport` only gets 92.20%, so the build might fail here. We need to either test more stuff or lower this number... or set `coverageFailOnMinimum` to false (probably a bad idea)

File: app/definition/ApiDefinitionFactory.scala (hunk 1 above)
Comment (jeremystone) [reply app/definition/ApiDefinitionFactory.scala]: Only there because initially it was set to v2, and it was decided to start at v1 (as this is a new API). But I left it in to show how other versions could be added. Happy to remove.

File: conf/application.conf (hunk 2 above)
Comment (jeremystone) [reply conf/application.conf]: Good question. Does not appear to be used.

File: project/CodeCoverageSettings.scala (hunk 3 above)
Comment (jeremystone) [reply project/CodeCoverageSettings.scala]: Will take a look at this.
